android-build/README.md

```diff
@@ -0,0 +1,3 @@
+# Android applications build
```

Comment:
Make this readme more useful try adding some simple usage instructions.
e.g.
`docker run --rm=true dubit/android-build -v path/to/project:/usr/src/app SomeBuildCommand`